src/sql/src/plan/scope.rs

```diff
@@ -379,7 +384,7 @@ impl Scope {
                 }
                 if item.table_name.as_ref().map(|n| n.matches(table_name)) == Some(true) {
                     seen_at_level = Some(lat_level);
-                    item.column_name == *column_name
+                    true
                 } else {
                     false
```

Comment:
If possible, it would be nice to get a test case for this and the other false scenario. One where a column with the same name exists, but it isn't suggested in the similar list.

Reply:
I _think_ the new test case I added covers this, not totally sure though